libft/sources/strings/ft_split_skip_delim_utils.c

```diff
@@ -6,18 +6,60 @@
 /*   By: taretiuk <taretiuk@student.42.fr>          +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2024/10/17 17:58:31 by taretiuk          #+#    #+#             */
-/*   Updated: 2024/11/26 15:12:40 by taretiuk         ###   ########.fr       */
+/*   Updated: 2024/12/10 15:45:33 by taretiuk         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "strings_internal.h"
 
-bool	is_in_quotes(const char **p, const char ex, int *in_quotes)
+bool	ft_is_quote(const char p, t_skip_chars ex_arr)
```

Comment:
please don't call this ft_, since this is an internal function, and ft_ prefix is for functions for external users of the library

Reply:
i dont use this function, so i deleted it